tests/plugins/test_bump_release.py

```diff
@@ -187,3 +190,34 @@ def ssl_login(self, cert, ca, serverca, proxyuser=None):
             assert 'Release' not in parser.labels
         else:
             assert parser.labels['Release'] == next_release['expected']
+
+    @pytest.mark.parametrize('base_release,builds,expected', [
+        ('42', [], '42.1'),
+        ('42', ['42.1', '42.2'], '42.3'),
```

Comment:
Could `base_release` already be in the form `'42.1'`?

Reply:
Would like to see a test case where `base_release` is e.g. `'42.1'`, or a comment explaining why it's not valid to test it.

Reply:
OK, will add:

```
        # No interpretation of the base release when appending - just treated as string                                                                    
        ('42.1', ['42.2'], '42.1.1'),
        ('42.1', ['42.1.1'], '42.1.2'),
```